web/packages/quick-editor/src/index.ts

```diff
@@ -0,0 +1,12 @@
+export type {
+	Scope,
+	ProfileUpdatedType,
+	OnProfileUpdated,
+	OnOpened,
+	QuickEditorCoreOptions
+} from './quick-editor-core';
+
+export type { QuickEditorOptions } from './quick-editor';
+
+export { default } from './quick-editor';
```

Comment:
If I remember correctly. We need to give it a name for the `UMD` build to work as below:

- Current: [The_UMD_global_variable](https://github.com/Automattic/gravatar/pull/14/files#diff-de5746fe258fbb418c3d465c4960dd234f39ff0ca3947b6eb5612274eb96c0cbR60).default
- Suggest: [The_UMD_global_variable](https://github.com/Automattic/gravatar/pull/14/files#diff-de5746fe258fbb418c3d465c4960dd234f39ff0ca3947b6eb5612274eb96c0cbR60).GravatarQuickEditor

```suggestion
export { default as QuickEditor } from './quick-editor';
```

Reply:
My idea here was to make the `GravatarQuickEditor` available as the default module, allowing developers to import it like `import GravatarQuickEditor from '@gravatar-com/quick-editor';
But that's not mandatory, I just thought it would make sense to have a default module.

Reply:
I have changed it as you suggested above.